index/bleve/query.go

```diff
@@ -6,8 +6,8 @@ import (
 	"github.com/moira-alert/moira/index/mapping"
 )
 
-func buildSearchQuery(filterTags, searchTerms []string, onlyErrors bool) query.Query {
-	if !onlyErrors && len(filterTags) == 0 && len(searchTerms) == 0 {
+func buildSearchQuery(filterTags, searchTerms []string, onlyErrors, needSearchByCreatedBy bool, createdBy string) query.Query {
```

Comment:
Maybe pass SearchOptions as a param?

Reply:
Changed the code to allow to pass `searchOptions` to parameters